### PR TITLE
Changes made

### PR DESCRIPTION
**Pull request by tswanatlassian**
Making changes to a file, commenting, then editing the file to discover what happens to said file.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Kirby is a file-based CMS.
 Easy to setup. Easy to use. Flexible as hell.
 
-## Trial
+## Kirby Trial
```
Comment (tswanatlassian) [inline readme.md]: Do we even need this at all?

Comment (timswn) [inline readme.md]: More to the point, do we need any of this?

File: readme.md [hunk 2]
```diff
@@ -3,12 +3,6 @@
 Kirby is a file-based CMS.
 Easy to setup. Easy to use. Flexible as hell.
 
-## Trial
-
-You can try Kirby on your local machine or on a test
-server as long as you need to make sure it is the right
-tool for your next project.
-
 ## Buy a license
```
Comment (timswn) [inline readme.md]: On second thought, let's get rid of this one too.

**Comment by timswn** [comment]
Want to add some activity here and there.

File: readme.md [hunk 3]
```diff
@@ -3,11 +3,7 @@
 Kirby is a file-based CMS.
```
Comment (tswanatlassian) [inline readme.md]: I think we probably need to change this in place of a more rounded description.

File: readme.md [hunk 4]
```diff
@@ -3,11 +3,7 @@
...
 Easy to setup. Easy to use. Flexible as hell.
 
-## Trial
-
-You can try Kirby on your local machine or on a test
-server as long as you need to make sure it is the right
-tool for your next project.
+#### I'm not going to get rid of this, but rather add some lines to change the position of the comment
 
 ## Buy a license
```
Comment (tswanatlassian) [inline readme.md]: Buy a license

File: readme.md (hunk 3 above)
Comment (timswn) [inline readme.md]: Agreed. Will update with time.

**Pull request merged**